services/QuillLMS/client/app/bundles/Comprehension/components/studentView/container.tsx

```diff
@@ -106,6 +114,59 @@ export class StudentViewContainer extends React.Component<StudentViewContainerPr
     });
   }
 
+  defaultHandleFinishActivity = () => {
+    this.specifiedActivitySessionUID();
```

Comment:
you aren't saving the output of this call in a variable so I think it can be removed

Reply:
Good catch.